zeroauthority-dao/za-website
# FIX: Results Page

## This Branch:

- adds `serviceId` dependency to `useEffect`

## Test:

- `npm run dev`
- navigate to `localhost:3000`
- click any service button
- click through various service buttons
- confirm users update on each selection
